Sources/Turf/JSON.swift

```diff
@@ -38,27 +38,22 @@ public enum JSONValue: Equatable {
      - parameter number: An integer. JSON does not distinguish numeric types of different precisions, so the integer is stored as a floating-point number.
      */
     public init<Source>(_ number: Source) where Source: BinaryInteger {
-        self = .number(Double(number))
+        self.init(Double(number))
     }
```

Comment:
Would it be better to extend `BinaryInteger` to conform to `JSONValueConvertible`?

Reply:
(and remove this init method?)

Reply:
No, unfortunately that isn’t possible:

```
/path/to/turf-swift/Sources/Turf/JSON.swift:233:1: Extension of protocol 'BinaryInteger' cannot have an inheritance clause
```